lib/analytics.js

```diff
@@ -592,10 +589,18 @@ Analytics.prototype._callback = function(fn) {
  */
 
 Analytics.prototype._invoke = function(method, facade) {
+  var self = this;
   this.emit('invoke', facade);
 
   each(function(integration, name) {
     if (!facade.enabled(name)) return;
+    var failedInitializations = self.failedInitializations || [];
```

Comment:
A bit confused on the placement of this (since it will be overwritten for each integration). Would this be better served after the this.emit()?

Reply:
youre right we should put the accessor outside the each loop. However, nothing will be overwritten. Not sure what you mean there.